packages/big-design/src/components/Form/Label/Label.tsx

```diff
@@ -1,7 +1,14 @@
 import React from 'react';
 
-import { StyledLabel } from './styled';
+import { StyledLabel, StyledSpan } from './styled';
 
-export type LabelProps = React.LabelHTMLAttributes<HTMLLabelElement>;
+export interface LabelProps extends React.LabelHTMLAttributes<HTMLLabelElement> {
+  renderOptional?: boolean;
+}
 
-export const Label: React.FC<LabelProps> = ({ className, style, ...props }) => <StyledLabel {...props} />;
+export const Label: React.FC<LabelProps> = ({ children, className, renderOptional, style, ...props }) => (
+  <StyledLabel {...props}>
+    {children}
+    {renderOptional && <StyledSpan> (optional)</StyledSpan>}
```

Comment:
🍹Why did you end up going with a styled `span` instead of css `content: "(optional)";`?

Reply:
Didn't even think about it, tbh. I seem to forget about this way of rendering extra text. Seems like a good idea.

Reply:
Any idea how to test pseudoelements in jest? Can't seem to get it to work.

Reply:
Beat me to suggesting `toHaveStyleRule`!